### PR TITLE
Use helper to generate behavior config array.

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: tests/comparisons/Model/testGetBehaviors.php [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+ * @method \App\Model\Entity\Post patchEntity(\Cake\Datasource\EntityInterface $entity, array $data, array $options = [])
+ * @method \App\Model\Entity\Post[] patchEntities($entities, array $data, array $options = [])
+ * @method \App\Model\Entity\Post findOrCreate($search, callable $callback = null, $options = [])
+ *
+ * @mixin \Cake\ORM\Behavior\CounterCacheBehavior
+ * @mixin \Cake\ORM\Behavior\TranslateBehavior
+ */
+class PostsTable extends Table
+{
+
```
Comment (dereuromark) [inline tests/comparisons/Model/testGetBehaviors.php]: We should remove that newline if possible (as per psr2). Or the generated code will always have to be sniffer fixed afterwards.

Comment (ADmad) [reply tests/comparisons/Model/testGetBehaviors.php]: Our coding standard does not report any error for this new line.

Comment (ADmad) [reply tests/comparisons/Model/testGetBehaviors.php]: I checked, neither does PSR2.

Comment (dereuromark) [reply tests/comparisons/Model/testGetBehaviors.php]: But it should, it states that there is no newline at beginning or end.

File: src/View/Helper/BakeHelper.php [hunk 2]
```diff
@@ -78,7 +78,19 @@ public function stringifyList(array $list, array $options = [])
...
             if (!is_numeric($k)) {
-                $v = "'$k' => $v";
+                $nestedOptions = $options;
+                if ($nestedOptions['indent']) {
+                    $nestedOptions['indent'] += 1;
+                }
+                if (is_array($v)) {
+                    $v = sprintf(
+                        "'%s' => [%s]",
+                        $k,
```
Comment (markstory) [inline src/View/Helper/BakeHelper.php]: Do we need to escape possible `'` in `$k`?

Comment (ADmad) [reply src/View/Helper/BakeHelper.php]: I don't see anyone using single quotes in config key names.

Comment (markstory) [reply src/View/Helper/BakeHelper.php]: Ok.

**Pull request merged**